resnet/cifar_input.py

```diff
@@ -101,7 +109,7 @@ def build_input(dataset, data_path, batch_size, mode):
   labels = tf.reshape(labels, [batch_size, 1])
   indices = tf.reshape(tf.range(0, batch_size, 1), [batch_size, 1])
   labels = tf.sparse_to_dense(
-      tf.concat(1, [indices, labels]),
```

Comment:
if you don't care about being able to run it in previous version of TF, I can change it to that